www/front_src/src/Resources/index.test.tsx

```diff
@@ -1054,4 +1058,28 @@ describe(Resources, () => {
       }),
     );
   });
+
+  it('leaves search help tooltip when the search input filled ', async () => {
```

Comment:
What are we testing here exactly? that the search input is still visible even if the help tooltip is displayed?

Reply:
that the tooltip is not closed when searching.
Cause it will be the main use case for user who want to get this help when typing a search